widget/table.go

```diff
@@ -244,7 +244,7 @@ func (t *tableRenderer) Layout(s fyne.Size) {
 }
 
 func (t *tableRenderer) MinSize() fyne.Size {
-	return t.cellSize
+	return t.t.scroll.MinSize().Max(t.cellSize.Add(fyne.NewSize(theme.Padding(), 0)))
```

Comment:
If we are leaving space for the indicators then table needs to have the same on the vertical direction

Reply:
Done